enclave_wrangler/objects_api.py

```diff
@@ -464,7 +464,19 @@ def fetch_cset_and_member_objects(
         cset_versions: List[Dict] = fetch_objects_since_datetime('OMOPConceptSet', since, verbose) or []
     # - Filter any old drafts from sourceApplicationVersion 1.0. Old data model. No containers, etc. See also:
     #  https://github.com/jhu-bids/TermHub/actions/runs/6489411749/job/17623626419
-    cset_versions = [x for x in cset_versions if x['properties']['conceptSetNameOMOP']]
+    cset_versions = [x for x in cset_versions if x and x['properties']['conceptSetNameOMOP']]
```

Comment:
## todo?: Best way to return info about discarded drafts?
The way that this function `fetch_cset_and_member_objects()` returns information about discarded drafts is purely implicit. If it receives `codeset_ids` to fetch, then if the draft is discarded, that cset will simply not be returned.

The fetch failures script knows about this and handles accordingly. But perhaps doing this in some more explicit way, such as adding a `discarded_drafts` key to the object/dict returned by the func would be nice.